nd4j/nd4j-backends/nd4j-backend-impls/nd4j-cuda/pom.xml

```diff
@@ -112,7 +112,6 @@
                         <includePath>${libnd4jhome}/include/indexing</includePath>
                     </includePaths>
                     <linkPaths>
-                        <linkPath>${libnd4jhome}/blasbuild/cuda-${cuda.version}/blas</linkPath>
```

Comment:
Hm, I would rather keep CUDA version, just in case if someone want to build several CUDA backends on same machine...
But, I may be wrong...

Reply:
That was just a build directory created by libnd4j, but it won't use it anymore. It's all in "cuda" now like it was before I tried making this work with symbolic links.

Reply:
Yeap, I remember that, ok then.